libs/shared/metrics/glean/src/registry/subplat-backend-metrics.yaml

```diff
@@ -1,3 +1,390 @@
 ---
 # Schema
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+account:
+  user_id_sha256:
+    type: string
+    description: |
+      A hex string of a sha256 hash of the account's uid
```

Comment:
Perhaps chat with Chelsea first, but we've been cleared to use the plain FxA uid now.